irteus/irtutil.l

```diff
@@ -368,6 +368,19 @@
           (reverse ret))
       (close strm))))
 
+(defun make-robot-model-from-name (name &rest args)
+  "make a robot model from string: (make-robot-model \"pr2\")"
+  (let ((name (string-downcase name)))
+    (let ((klasses (flatten (send robot-model :hierarchy))))
+      (dolist (klass klasses)
+        (let ((klass-name (string-downcase (send klass :name))))
+        (if (or (string= klass-name name)
```

Comment:
I'd like to keep searching rules simple.
Why isn't it sufficient to search [robot-name]-modef classl?
Which robot has class name like [robot-name]-robot-model?

Moreover, currently `pr2-sensor-robot` cannot be searched by this.
However, in this case, we may rename:
- `pr2-robot` -> `pr2-without-sensor-robot` (converted from euscollada)
- `pr2-sensor-robot` -> `pr2-robot` (defined in the last part of pr2.l)